geopandas/geodataframe.py

```diff
@@ -777,6 +789,42 @@ def _to_geo(self, **kwargs):
 
         return geo
 
+    def to_wkb(self):
```

Comment:
```suggestion
    def to_wkb(self, hex=False):
```

The original array method has `hex` keyword to control. We should expose it here as well. You can copy docstring from pygeos https://pygeos.readthedocs.io/en/latest/io.html#pygeos.io.to_wkb

Reply:
I had put this in originally, but took it out. I think the reason is that I had also included a `hex` keyword arg in the `from_wkb` methods, but the tests failed because pygeos didn't support loading from wkb hex. Then when I took that out I must have gone a little overboard and removed more than I needed to.